apps/ensadmin/src/app/quickstarts/page.tsx

```diff
@@ -0,0 +1,574 @@
+"use client";
+
+import { ConnectionSelector } from "@/components/connections/connection-selector";
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components/ui/accordion";
+import {
+  Card,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import { CodeBlock } from "@/components/ui/code-block";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
+import {
+  Database,
+  ExternalLink,
+  FileSearch,
+  PackagePlus,
+  PlayCircle,
+} from "lucide-react";
+import { useRouter, useSearchParams } from "next/navigation";
+import { Suspense, useCallback } from "react";
+
+const QUERY_EXAMPLES = [
+  {
+    id: "domain-by-id",
+    name: "Domain by ID",
+    description: "Query a specific domain using its node ID",
+    graphql: `query GetDomain($id: ID!) {
```

Comment:
I like how we have a dedicated page for that quick start experience 🚀 

I can help with providing a list of useful queries if you like 🙃